### ISSUE TITLE
[1.4.1] - GuideAPI needs to be downgraded

### ISSUE THREAD

**Issue by draeath** [Vaygrim/vcs3#45]
I'm trying to get a specific build, but the Sanguimancy book is almost unreadable. [The fix is apparently to downgrade GuideAPI "by 1 build."](https://github.com/Tombenpotter/Sanguimancy/issues/110)

**Comment by Vaygrim** [comment]
Yeah we're going to need some more specifics before we can correct anything. "By 1 build" is fairly arbitrary.

**Comment by draeath** [comment]
OK, I got something more specific: one released version.

This means 1.0.1-23, which is still beta. Looking back at the Guide-API commit logs for the 1.7.10 branch, I only see one commit between the dates that 1.0.1-23 and 1.0.1-29 - this [one specifically](https://github.com/TeamAmeriFrance/Guide-API/commit/320b67ac2e655778419ba7308eb14ac489da31ca). It _sounds_ like a minor issue, at least based on the commit message.

_UNFORTUNATELY_ it did not solve the issue, so trying to get something more specific _again._ On the bright side, it didn't cause any crashes or such (as expected based on the changes)
